test/data/inner_module.js

```diff
@@ -1 +1 @@
-window.ok( true, "evaluated: inner module with src" );
```

Comment:
Whoa, that kind of makes it clear we were depending on an `ok()` function in global scope. Scary.